installer/bundle/bundle_skel.sh

```diff
@@ -759,11 +732,21 @@ then
     echo "Failed: could not extract the install bundle."
     cleanup_and_exit ${STATUS}
 fi
+[ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+[ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+./bundles/${SCX_INSTALLER} --extract $FORCE $DEBUG
```

Comment:
It depends; what should we do, from a design perspective? I hadn't heard of the --extract use case until I started looking at this last week.

From your comments, I think the user can call --extract on the SCX installer separately. I will make this change.